bugalink-frontend/components/externalLogin/index.tsx

```diff
@@ -0,0 +1,23 @@
+import RoundedButton from '../buttons/Rounded';
+import Facebook from '../../public/icons/Login/facebook.svg';
+import Twitter from '../../public/icons/Login/twitter.svg';
+import Google from '../../public/icons/Login/google.svg';
+
+export default function ExternalLogin() {
+  return (
+    <div>
+      <div className="flex flex-row items-center justify-center space-x-8">
+        <RoundedButton
+          link="#"
+          Icon={<Facebook className="ml-1 h-8 w-8 scale-125" />}
+        />
+        <RoundedButton link="#" Icon={<Twitter className="ml-1.5 h-7 w-7" />} />
+        <RoundedButton
+          link="#"
+          Icon={<Google className="ml-1.5 h-7 w-7 fill-red" />}
+        />
+      </div>
+      <p className="font-bold text-light-gray">o usa tu cuenta de correo</p>
```

Comment:
This is silly, but if in the end "or use your email account" is not an external login, shouldn't it go outside this component? I understand that it is for style reasons more than anything else.